<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>OCC::IgnoreListTableWidget</class>
    <widget class="QWidget" name="OCC::IgnoreListTableWidget">
        <property name="windowTitle">
            <string notr="true">IgnoreListTableWidget</string>
        </property>
        <layout class="QVBoxLayout" name="verticalBoxLayout">
            <property name="bottomMargin">
                <number>40</number>
            </property>
            <item>
                <layout class="QHBoxLayout" name="horizontalBoxLayout">
                    <property name="spacing">
                        <number>24</number>
                    </property>
                    <property name="bottomMargin">
                        <number>15</number>
                    </property>
                    <item>
                        <widget class="QTableWidget" name="tableWidget">
                            <property name="enabled">
                                <bool>true</bool>
                            </property>
                            <property name="selectionMode">
                                <enum>QAbstractItemView::SingleSelection</enum>
                            </property>
                            <property name="selectionBehavior">
                                <enum>QAbstractItemView::SelectRows</enum>
                            </property>
                            <property name="columnCount">
                                <number>2</number>
                            </property>
                            <column>
                                <property name="text">
                                    <string>Pattern</string>
                                </property>
                            </column>
                            <column>
                                <property name="text">
                                    <string>Allow Deletion</string>
                                </property>
                            </column>
                        </widget>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalButtonLayout">
                            <property name="spacing">
                                <number>15</number>
                            </property>
                            <property name="alignment">
                                <enum>Qt::AlignTop</enum>
                            </property>
                            <item>
                                <widget class="QPushButton" name="addPushButton">
                                    <property name="enabled">
                                        <bool>true</bool>
                                    </property>
                                    <property name="text">
                                        <string>Add</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="removePushButton">
                                    <property name="enabled">
                                        <bool>true</bool>
                                    </property>
                                    <property name="text">
                                        <string>Remove</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="removeAllPushButton">
                                    <property name="text">
                                        <string>Remove all</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLabel" name="descriptionLabel">
                    <property name="enabled">
                        <bool>true</bool>
                    </property>
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="text">
                        <string />
                    </property>
                    <property name="textFormat">
                        <enum>Qt::PlainText</enum>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>